pysatMissions/instruments/_core.py

```diff
@@ -22,3 +23,22 @@ def _download(date_array, tag, sat_id, data_path=None):
     """ Data is simulated so no download routine is possible. Simple pass
     function"""
     pass
+
+
+def _get_times(fnames, sat_id):
+    """Construct list of times for simulated instruments"""
```

Comment:
I am a bit concerned about spreading the use of sat_id as the number of samples. It is how pysat currently is, so I suppose outside packages have to work with what pysat offers now. Could be worth noting in the docstring here.

Reply:
Also, couldn't we just import the version in pysat? Or, in the future?

Reply:
Ideally, yes. The pysat simplification ideas came after I put these together.  Once that is finalized, I can start updating this.